addOns/automation/src/test/java/org/zaproxy/addon/automation/jobs/ActiveScanConfigJobUnitTest.java

```diff
@@ -296,6 +301,64 @@ void shouldVerifyAndApplyInputVectors() {
         verify(param).setTargetParamsEnabledRPC(40);
     }
 
+    @Test
+    void shouldVerifyAndApplyExcludes() throws Exception {
```

Comment:
Worth adding a test where the excludes are not defined and it clears the existing ones.